test/current_directory.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:io';
+
+import 'package:path/path.dart';
+
+final Directory testDirectory = Directory(join(
```

Comment:
Filename is weird/misleading for me. What do you think about `test_directory`\`test_directories`?

Reply:
I'll call it directories since `projectDirectory` is the one used and derived from `testDirectory`